modules/rest-api/src/blaze/rest_api/routes.clj

```diff
@@ -97,127 +98,129 @@
   (when-let
    [{:blaze.rest-api.resource-pattern/keys [interactions]}
     (u/resolve-pattern resource-patterns structure-definition)]
-    (cond->
-     [(str "/" name)
-      {:fhir.resource/type name}
-      [""
-       (cond-> {:name (keyword name "type")}
-         (contains? interactions :search-type)
-         (assoc :get {:interaction "search-type"
-                      :middleware [[wrap-db node db-sync-timeout]
+   (cond->
+    [(str "/" name)
+     {:fhir.resource/type name}
+     [""
+      (cond-> {:name (keyword name "type")}
+        (= name "Binary")
+        (assoc :response-type :binary)
+        (contains? interactions :search-type)
+        (assoc :get {:interaction "search-type"
+                     :middleware [[wrap-db node db-sync-timeout]
+                                  wrap-link-headers]
```

Comment:
I assume that you did not rebase properly onto the current main. thats why you have so much change here.